lua/plenary/busted.lua

```diff
@@ -214,7 +233,9 @@ after_each = mod.after_each
 clear = mod.clear
 assert = require "luassert"
 
-mod.run = function(file)
+mod.run = function(file, opts)
+  _PlenaryBustedOpts = vim.tbl_deep_extend("force", {}, opts or {})
```

Comment:
doing a global is technically fine because busted is already run in another process. But can't we do 

```lua
-- top of the file
local busted_opts

--- ......

mod.run = function(file, opts)
  busted_opts = vim.F.if_nil(opts, {})
-- ...
end
```

Reply:
Done, thanks! Although I did a `vim.F.if_nil(opts, {}, opts)` (3rd arg) 👍